training/index.html

```diff
@@ -0,0 +1,75 @@
+---
+layout: default
+title: OMERO Training
+---     
+        {% include navbar-sub-omero.html %}
+        
+        <div class="callout large primary" id="bg-image-omero">
+            <div class="row column text-center">
+                <h1>OME Training</h1>
+                <p>
+                    OME wants to promote understanding of our products with practical
+                    workshops in the community.
+                </p>
+            </div>
+        </div>
+        
+
+        <hr class="whitespace">
+        <div class="row column">
+            <p>
+                The OME team is keen to provide outreach workshops in the community to
+                demonstrate the features of our tools within imaging workflows. The training
+                materials we develop are freely available along with the image data and
+                setup procedures used.
+            </p>
+            <p>
+                Our workshops range in length from a 1-hour overview of OMERO up to a full day.
+                For more detailed coverage, a 2-day workshop can be used to provide an introduction
+                to OMERO on one day and more advanced analysis, developer or sysadmin features
+                on the next.
+            </p>
+        </div>
+
+        <div class="row">
+            <div class="small-12 medium-3 columns">
```

Comment:
This card looks larger on my screen.

Reply:
That's because the text is longer. All those boxes expand to fit the text. We could try making the text a tiny bit shorter.